crates/turborepo-lib/src/package_graph/mod.rs

```diff
@@ -38,10 +38,13 @@ impl WorkspaceInfo {
     }
 }
 
+type PackageName = String;
+type PackageVersion = String;
+
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Ord, PartialOrd)]
 pub struct Package {
```

Comment:
A quick search makes it look like this was only used in external deps set. I vote we just drop it.

Reply:
I added it mostly for my clarification because a `HashMap<String, String>` is pretty opaque

Reply:
Oops, I meant the `Package` struct not the type aliases.